docs/community/release-process/smart-contracts.mdx

```diff
@@ -148,7 +148,7 @@ Submit the autogenerated upgrade proposal to the Governance contract for review
 
 ```bash
 # resultant proposal ID should be communicated publicly
-celocli governance:propose --deposit 100e18 --from $YOUR_ADDRESS --jsonTransactions "proposal.json"
+celocli governance:propose --deposit 100e18 --from $YOUR_ADDRESS --jsonTransactions "proposal.json" --descriptionURL https://gist.github.com/yorhodes/46430eacb8ed2f73f7bf79bef9d58a33
```

Comment:
Usually these URLs point to a CGP in the CGP's repo, so maybe use this as example URL: `https://github.com/celo-org/governance/blob/main/CGPs/cgp-0055.md`

Reply:
Thanks, updated